hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/multitenant/TestMultiTenantVolume.java

```diff
@@ -138,7 +138,8 @@ private ObjectStore getStoreForAccessID(String accessID) throws Exception {
     // Manually construct an object store instead of using the cluster
     // provided one so we can specify the access ID.
     RpcClient client = new RpcClient(conf, null);
-    client.setTheadLocalS3Auth(new S3Auth("unused1", "unused2", accessID));
+    client.setTheadLocalS3Auth(
+        new S3Auth("unused1", "unused2", accessID, accessID));
```

Comment:
This is confusing as to why we are setting the accessID twice. Need some comment here.

Reply:
done!